tools/hasher/pom.xml

```diff
@@ -50,7 +50,11 @@
         </dependency>
         <dependency>
             <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-simple</artifactId>
+            <artifactId>slf4j-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
```

Comment:
nice cleanup 😄

Reply:
It is not so funny actually. See the chat in slack. To be able test things written to `System.out`, we need `logback-classic`, as the `simple-logger` cannot be intercepted.
Romain suggested to refactor the hasher utility a bit further, and I would like to do that in another PR.